.github/workflows/compile.yml

```diff
@@ -7,19 +7,17 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
     - name: Install packages
-      run: sudo apt install -y opam zlib1g-dev pkg-config libgmp-dev z3 device-tree-compiler
+      run: sudo apt install -y opam zlib1g-dev pkg-config libgmp-dev device-tree-compiler curl
```

Comment:
I think we should be able to get rid of opam now as well.

Reply:
I think it's still needed to build the OCaml version of the emulator which hasn't been removed yet. Actually I may need to keep the `opam init`, but I'll see what happens.

Reply:
We might be able to just install OCaml directly in that case and avoid `opam init`.